integration-cli/docker_api_network_test.go

```diff
@@ -64,7 +65,16 @@ func (s *DockerSuite) TestAPINetworkCreateCheckDuplicate(c *check.C) {
 	c.Assert(isNetworkAvailable(c, name), checker.Equals, true)
 
 	// Creating another network with same name and CheckDuplicate must fail
-	createNetwork(c, configOnCheck, http.StatusConflict)
+	isOlderAPI := versions.LessThan(testEnv.DaemonAPIVersion(), "1.34")
+	expectedStatus := http.StatusConflict
+	if isOlderAPI {
+		// in the early test code it uses bool value to represent
+		// whether createNetwork is expected to fail fo nor.
```

Comment:
@AntaresS little typo. We should also explain that this will check that the status is NOT equal to http.StatusCreated.

Reply:
Done.